index.html

```diff
@@ -0,0 +1,329 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Portfolio</title>
+  <link rel="stylesheet" href="style.css" />
+</head>
+
+<body>
+
+
+  <section class="headline">
+    <div id="head-container">
+      <h1 class="head">
+        I'm John, <br />
+        Glad to see you!
+      </h1>
+      <p class="desc">
+        I’m a software developer! I can help you build a product, feature or
+        website Look through some of my work and experience. If you like what
+        you see and have a project you need coded, don’t hesitate to contact
+        me!
+      </p>
+      <h2 class="connect">LET’S CONNECT</h2>
+
+      <ul class="social-media-icons">
+        <li>
+          <a href="#" target="_blank">
+            <img src="./images/twitter.png" alt="twitter" />
+          </a>
+        </li>
+        <li>
+          <a href="#" target="_blank">
+            <img src="images/Linkedin.png" alt="linkedIn" />
+          </a>
+        </li>
+        <li>
+          <a href="#" target="_blank">
+            <img src="images/medium.png" alt="medium" />
+          </a>
+        </li>
+        <li>
+          <a href="#" target="_blank">
+            <img src="images/github.png" alt="git" />
+          </a>
+        </li>
+        <li>
+          <a href="#" target="_blank">
+            <img src="images/angelite.png" alt="angel" />
+          </a>
+        </li>
+      </ul>
+    </div>
+  </section>
+
+  <main class="grid-container">
+    <section class="works-card-tonic">
```

Comment:
- [ ] Kindly consider wrapping the whole cards container in a section tag and each project card in a div to make your code look more professional after that consider making sure that the space between two cards is the same as the one given on Figma to make your project an exact duplicate of the given one check the images below 
- your design 👇 
<img width="281" alt="Screen Shot 2023-05-05 at 20 17 04" src="https://user-images.githubusercontent.com/60146030/236537070-24f63b98-60f7-419d-84b4-b87cef312733.png">

-Figma design 👇 
<img width="230" alt="Screen Shot 2023-05-05 at 20 17 38" src="https://user-images.githubusercontent.com/60146030/236537178-83a54a6d-ce1b-4bbd-950c-f3d13fcb0841.png">

Reply:
ok. I change the height in this grid.
![image](https://user-images.githubusercontent.com/35693330/236592671-8318a30f-8afa-44ad-b70b-f90aecdac6c5.png)

![image](https://user-images.githubusercontent.com/35693330/236592703-bc3562fb-f056-4434-8e24-c0ab23727804.png)


![image](https://user-images.githubusercontent.com/35693330/236592714-a093cbf6-bea2-450d-a63f-edeb4c819694.png)

![image](https://user-images.githubusercontent.com/35693330/236592724-37279814-b1a7-4c11-8b65-cf51f58c83d6.png)


![image](https://user-images.githubusercontent.com/35693330/236592729-f93ab4e0-3f86-4c50-b8f7-65f6ccf504b5.png)